.github/actions/gcp-asset-inventory-ci/action.yml

```diff
@@ -35,9 +35,10 @@ runs:
       run: |
         ./cloudbeat -c deploy/asset-inventory/cloudbeat-gcp-asset-inventory.yml -d '*' &
 
-    - name: Wait for cloudbeat to send some events
-      shell: bash
-      run: sleep 20
+    - name: Wait for data
+      uses: ./.github/actions/wait-for-es-data
+      with:
+        index: .ds-logs-cloud_asset_inventory.asset_inventory-*
```

Comment:
![Screenshot 2024-09-15 at 14 36 15](https://github.com/user-attachments/assets/87f12dca-ed0a-4e4b-8d14-590b2df37533)


in this case it took 26~ seconds